src/web/environments/config.template.ts

```diff
@@ -16,4 +16,11 @@ export const config: any = {
    * The support email shown to the user in various pages of the web application.
    */
   supportEmail: 'teammates@comp.nus.edu.sg',
+
+  /**
+   * The public site key for the reCAPTCHA on the recover session links page.
+   * You can get a pair of keys from the Google reCAPTCHA website.
+   * e.g. "6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI" is a site key for test environments.
+   */
+  captchaSiteKey: '6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI',
```

Comment:
If this and/or the back-end secret key is not set, let's skip the CAPTCHA part? When developing locally there's little to zero need to have to verify CAPTCHA every time we want to test changes.

Reply:
Ok, I've done this for the front end (not sure whether it's the right way though...)

For the backend, we'll need to add the check in `SessionLinksRecoveryAction#execute()` in Junming's PR